README.md

```diff
@@ -104,12 +113,12 @@ Body (example):
 
 ```json
 {
-    "message": "Service removal has failed"
+    "errors": ["Service removal has failed"]
 }
 ```
 
 ## List a given user's reservations
-### Endpoint: GET api/v1/user/:id/reservations
+### Endpoint: GET api/v1/users/:id/reservations
 #### Response
 Successful:
 Code: 200
```

Comment:
Good job with this documentation! as mentioned above, you can go a step further usering swagger)